test/hotspot/jtreg/runtime/CommandLine/OptionsValidation/common/optionsvalidation/JVMOptionsUtils.java

```diff
@@ -53,22 +53,22 @@ public class JVMOptionsUtils {
     private static final StringBuilder finalFailedMessage = new StringBuilder();
 
     /* Used to start the JVM with the same type as current */
-    static String VMType;
+    static String VMType = null;
```

Comment:
No need, `static`-s are implicitly initialized to `null`. Otherwise there is a style question why any other fields are not initialized! Maybe initialize it straight in `clinit`, like `GCType` is initialized.

Reply:
Right, static field with reference type has null as the default value. Took your other suggestion in the `clinit` and remove `= null` here.